misc/python/materialize/cli/mzcompose.py

```diff
@@ -18,9 +18,8 @@
 from pathlib import Path
 from typing import IO, List, Optional, Sequence, Text, Tuple
 
-from typing_extensions import NoReturn
-
 from materialize import errors, mzbuild, mzcompose, spawn, ui
+from typing_extensions import NoReturn
```

Comment:
Ops, I pushed an import order change in an unrelated file. What is the deal with those? I keep getting them every time I run bin/pylint

Reply:
I believe that this might be the `isort` call in `bin/pyfmt`